src/pages/instances/InstanceSnapshots.tsx

```diff
@@ -216,23 +209,22 @@ const InstanceSnapshots: FC<Props> = ({ instance }) => {
                   aria-label="Search for snapshots"
                 />
               </div>
-              <ConfigureSnapshotsBtn
+              <InstanceConfigureSnapshotsBtn
                 instance={instance}
                 className="u-no-margin--right"
-                onSuccess={onSuccess}
                 onFailure={onFailure}
+                onSuccess={onSuccess}
               />
-              <Button
-                appearance="positive"
+              <InstanceAddSnapshotBtn
+                instance={instance}
+                onSuccess={onSuccess}
+                onFailure={onFailure}
                 className="u-float-right"
-                onClick={() => setModalOpen(true)}
-              >
-                Create snapshot
-              </Button>
+              />
```

Comment:
I think those two buttons should also be disabled, when the project has restricted snapshot creation. Might be worth adding a tooltip or title text explaining why they are disabled. Or find a spot on the page to display the sentence, like in the empty state with disabled snapshots.
(same applies to the volume snapshots page)

Reply:
Yeah that makes sense, will add the help text and disable those buttons

Reply:
I think a warning for configuring snapshot settings when snapshot is restricted makes sense. Perhaps this can be a separate task as it may require some design input. Cool if I create a Jira task for this?

Reply:
Yep, thank you :+1:

Reply:
[task link](https://warthogs.atlassian.net/browse/WD-7885)